gef-ui/component-tests/partials/amendments/amendment-details-further-makers-input-required.component-test.ts

```diff
@@ -0,0 +1,105 @@
+import { FACILITY_TYPE, PORTAL_AMENDMENT_STATUS, ROLES } from '@ukef/dtfs2-common';
+import pageRenderer from '../../pageRenderer';
+import { AmendmentDetailsViewModel } from '../../../server/types/view-models/amendments/amendment-details-view-model';
+
+const page = 'partials/amendments/amendment-details.njk';
+const render = pageRenderer(page);
+
+const dealId = '6597dffeb5ef5ff4267e5044';
+const facilityId = '6597dffeb5ef5ff4267e5045';
+const amendmentId = '6597dffeb5ef5ff4267e5046';
+
+const users = [ROLES.MAKER, ROLES.CHECKER];
+
+users.forEach((user) => {
+  describe(`when the user is a ${user}`, () => {
+    describe(page, () => {
+      const previousPage = 'previousPage';
+      const exporterName = 'exporterName';
+      const facilityType = FACILITY_TYPE.CASH;
+      const userRoles = [user];
+      const submitAmendment = user.includes(ROLES.CHECKER);
+      const effectiveDate = '25/07/2025';
+      const banner = true;
+      const amendmentStatus = PORTAL_AMENDMENT_STATUS.FURTHER_MAKERS_INPUT_REQUIRED;
+      const canSubmitFacilityAmendment = user.includes(ROLES.MAKER) && amendmentStatus === PORTAL_AMENDMENT_STATUS.FURTHER_MAKERS_INPUT_REQUIRED;
```

Comment:
```suggestion
      const canSubmitFacilityAmendmentToUkef = user.includes(ROLES.MAKER) && amendmentStatus === PORTAL_AMENDMENT_STATUS.FURTHER_MAKERS_INPUT_REQUIRED;
```

Would suggest suffixing with `toUkef` to be concise with where it is being submitted.

Reply:
Can this please be applied across the PR.

Reply:
I think the role should be `CHECKER`.

Reply:
@abhi-markan why should I suffix this with ukef? I could to this canSubmitFacilityAmendmentToChecker if it is confusing, but this is when Maker submit an amendment to checker

Reply:
Apologies, I thought this was submitting to TFM, yes please can it be suffixed as per yoru suggestion `canSubmitFacilityAmendmentToChecker`. Thanks